domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -147,6 +153,16 @@ class CameraXCameraUseCase @Inject constructor(
         recording?.stop()
     }
 
+     override fun setFlashMode(flashModeStatus: FlashModeStatus){
+        val newFlashMode = when (flashModeStatus) {
```

Comment:
nit: you can inline the assignment here: `imageCaptureUseCase.flashMode = when (flashModeStatus) { ... } `